python/dgl/heterograph.py

```diff
@@ -1143,7 +1143,7 @@ def has_nodes(self, vids, ntype=None):
         --------
         has_node
         """
-        vids = utils.toindex(vids)
+        vids = utils.toindex(vids, self._graph.dtype)
```

Comment:
Add a new API `idtype` to heterograph too.

Reply:
Sure